Default window created

KGebski0036 opened the pull request.

KGebski0036 committed:
Default window created

marcinwozniak0 commented on CMakeLists.txt:
```diff
@@ -10,15 +10,32 @@ set(CMAKE_CXX_FLAGS "-g3 -pthread -fconcepts -Wall -pedantic")
+set(CMAKE_AUTORCC ON)
+
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Widgets LinguistTools REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets LinguistTools REQUIRED)
```
sety i findy wrzuciłbym na górę pliku, żeby nie nie rozdzielać tworzenia binarek

marcinwozniak0 commented on CMakeLists.txt:
```diff
@@ -10,15 +10,32 @@ set(CMAKE_CXX_FLAGS "-g3 -pthread -fconcepts -Wall -pedantic")
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Widgets LinguistTools REQUIRED)
```
Potrzebujemy tego Qt6 ?

marcinwozniak0 commented on CMakeLists.txt:
```diff
@@ -10,15 +10,32 @@ set(CMAKE_CXX_FLAGS "-g3 -pthread -fconcepts -Wall -pedantic")
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets LinguistTools REQUIRED)
+
+set(PROJECT_SOURCES
+        main.cpp
```
wyrównanie do PROJECT_SOURCES

KGebski0036 committed:
Cmake fixes

Remove Qt6 and Linguist to fix compilation error. sets and finds moved to the top of the code
